pdh_check/setup.py

```diff
@@ -26,7 +26,7 @@ def get_requirements(fpath):
         return f.readlines()
 
 
-CHECKS_BASE_REQ = 'datadog_checks_base'
+CHECKS_BASE_REQ = 'datadog_checks_base>10.2'
```

Comment:
I assume we are targeting `10.3` and up, but this would also match e.g. `10.2.1`, so we should probably change this to…

```suggestion
CHECKS_BASE_REQ = 'datadog_checks_base>=10.3'
```

Reply:
I think that'll have to be in a separate PR until datadog_checks_base is released.

Reply:
I removed the pin for now, we can add the pin back once released.